src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
-public class ColorSupplier {
+import java.util.Random;
+
+public class ColorSupplier extends Ball {
     public String getRandomColor() {
-        return null;
+        int random = new Random().nextInt(BallCollor.values().length);
```

Comment:
Creating a new instance of Random every time the method is called is not efficient. It's better to create a single instance of Random as a class-level variable and reuse it.